### PR TITLE
Add test that confirms the outdated packages are deleted

### PR DESCRIPTION
**Pull request by suricactus**

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2432 Deleting obsolete packages is impossible due to too strict regex](https://app.clickup.com/t/2192114/QF-2432)

**Review by mbernasocchi — APPROVED** [review]

File: docker-app/qfieldcloud/core/tests/test_packages.py [hunk 1]
```diff
@@ -655,3 +657,54 @@ def test_collaborator_via_team_can_package(self):
...
             )
+
+    def test_outdated_packaged_files_are_deleted(self):
+        cur = self.conn.cursor()
+        cur.execute("CREATE TABLE point (id integer, geometry geometry(point, 2056))")
+        self.conn.commit()
+        cur.execute(
+            "INSERT INTO point(id, geometry) VALUES(1, ST_GeomFromText('POINT(2725505 1121435)', 2056))"
+        )
+        self.conn.commit()
```
Comment (olivierdalang) [inline docker-app/qfieldcloud/core/tests/test_packages.py:668]: not too sure how this is needed in this test ? (or is simply expected by `project2.qgs` ?)

**Review by olivierdalang — COMMENTED** [review]
just to be sure, is this also fixing some bug ? (not sure if the changes on the regexes just improvements or actual fixes)

**Comment by suricactus** [comment]
@olivierdalang It is a real issue (not breaking) that was attached in the second comment from clickup and sentry, regarding the `packages` regex. The other two regex fixes were something I found after second inspection of the regexes, didn't cause any harm or logs.

File: docker-app/qfieldcloud/core/tests/test_packages.py (hunk 1 above)
Comment (suricactus) [reply docker-app/qfieldcloud/core/tests/test_packages.py:668]: it is inspected in `project2.qgs`.

File: docker-app/qfieldcloud/core/utils2/storage.py [hunk 2]
```diff
@@ -568,7 +568,10 @@ def get_stored_package_ids(project_id: str) -> Set[str]:
 def delete_stored_package(project_id: str, package_id: str) -> None:
     prefix = f"projects/{project_id}/packages/{package_id}/"
 
-    if not re.match(r"^projects/[\w]{8}(-[\w]{4}){3}-[\w]{12}/packages/\w+/$", prefix):
+    if not re.match(
```
[the remark this answers is not in the document]
Comment (suricactus) [reply docker-app/qfieldcloud/core/utils2/storage.py:574]: Done.

**Pull request merged**